http/headers/cross-origin-embedder-policy.json

```diff
@@ -31,7 +31,7 @@
               "version_added": false
             },
             "safari": {
-              "version_added": "preview",
+              "version_added": "15",
```

Comment:
This flag I've deliberately not copied to Safari iOS as it does not show on the simulator, nor my iPad.

Reply:
This is also fairly redundant once https://github.com/mdn/browser-compat-data/pull/14006 is in